### DISCUSSION TITLE
Fine tuning settings

### DISCUSSION THREAD

**Discussion by DapperDom** [NightscoutFoundation/xDrip#2626] [Q&A]
Hi,

Firstly, apologies, starting a few threads as have several burning questions and I'm finally getting time to post them, so kind of all in one go!

I've got a pretty reliable setup running now with Libre 2 over BT, but every now and again I'll miss a reading for 10-20 mins or so.

Not too dramatic, but I'd love to dial out those issues if I can, so...

It seems that whenever that happens there's an entry in the error log that reads:

DexCollectionService
Descriptor write did not callback! since: <datetime>

Does anyone know what might be causing that and/or what settings I should look at altering to affect it?

I've played around with all the Bluetooth Settings in 'Less Common Settings', but none of those seem to affect it (at least obviously).  Anywhere else to dig into, or is this just something that can happen from time to time and it's just "one of those things"?

Thanks!
Dom

**Comment by Navid200** [comment]
@tzachi-dar @psonnera
